api.go

```diff
@@ -9,9 +9,9 @@ import (
 	"sync"
 	"time"
 
-	"github.com/hashicorp/go-hclog"
+	hclog "github.com/hashicorp/go-hclog"
```

Comment:
I've not seen linters making this change before and this is a convention understood by the compiler. I don't think it's necessary.

Reply:
Yeah that can be reverted, I guess it's something about explicit vs implicit in the linter. It's functional both ways too so no worries